org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -742,7 +747,7 @@ abstract class AbstractLanguageServerTest implements Endpoint {
 			textDocument = new TextDocumentIdentifier(fileInfo.uri)
 		]).get()
 		
-		assertEqualsStricter(configuration.expectedFoldings, String.join("", foldings.map[toString]));
+		assertEqualsStricter(configuration.expectedFoldings, String.join("", foldings.map[toExpectation]));
```

Comment:
Instead of using `String.join("", foldings.map[toExpectation])`, I would prefer using `foldings.toExpectation`

Reply:
Thanks for fixing it @Darkgaja , it looks really good now!